js/navigation.jquery.js

```diff
@@ -419,6 +423,26 @@ class Navigation {
       $nextNav.find('> a').focus();
     }
   }
+
+  static tryNextNav($focus) {
+    this.tryNextPrevNav($focus.parents('.navbar'), true);
```

Comment:
From a style POV can we use `Navigation` rather than `this` when we're calling static methods as using `this` in a static method is code smell?

Reply:
Yes, I'm surprised IDEA didn't do this. I had a look to see if there was a relevant ESLint rule we could add, but didn't spot anything relevant